tests/regression/lp-1805838/task.yaml

```diff
@@ -6,21 +6,48 @@ details: |
 prepare: |
     #shellcheck source=tests/lib/dirs.sh
     . "$TESTSLIB"/dirs.sh
-    mount --bind /bin/false "$LIBEXECDIR/snapd/snap-seccomp"
-    test -d "$SNAP_MOUNT_DIR/core" && mount --bind /bin/false "$SNAP_MOUNT_DIR/core/current/usr/lib/snapd/snap-seccomp"
-    test -d "$SNAP_MOUNT_DIR/snapd" && mount --bind /bin/false "$SNAP_MOUNT_DIR/snapd/current/usr/lib/snapd/snap-seccomp"
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local network-consumer
+
+    version_info="$($LIBEXECDIR/snapd/snap-seccomp version-info)"
+    if [ -d "$SNAP_MOUNT_DIR/core" ]; then
+        # core snap is not repacked with built snapd on core18
```

Comment:
The comment talks about `core` and `core18` but the presence of `core` is being tested. I don't see how that test is sufficient.

Reply:
In the end I reverted that last patch. Feels like there's too much things to account for to reliably interrogate exactly the right snap-confine.